libraries/AP_Scripting/examples/qnh_alt.lua

```diff
@@ -0,0 +1,51 @@
+--[[
+    example that sends QNH altitude to GCS as a NAMED_VALUE_FLOAT QNG_ALT_FT in feet
+
+    Note: operator must be QNH_PRESSURE in hPa before each flight!
```

Comment:
must "set"

Reply:
s/be "Note: operator must **set** QNH_PRESSURE in hPa before each flight!"

Reply:
fixed